core/arch/arm/plat-imx/a9_plat_init.S

```diff
@@ -91,10 +121,14 @@ UNWIND(	.fnstart)
 	mov_imm r0, 0x00004000
 	write_sctlr r0
 
-	mov_imm r0, 0x00000041
+#ifdef CFG_MX6SLL
```

Comment:
It would be nice with an updated description of the values put into this register.